redfish-core/lib/assembly.hpp

```diff
@@ -514,8 +514,11 @@ inline void checkAssociation(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
         return;
     }
 
-    aResp->res.jsonValue["Assemblies"] = nlohmann::json::array();
-    aResp->res.jsonValue["Assemblies@odata.count"] = 0;
+    if (!setLocationIndicatorActiveFlag)
```

Comment:
This is an atypical control flow.. :/ 
I am good with this only because we already have this downstream